app/index.html

```diff
@@ -130,6 +131,7 @@ <h4 class="modal-title" id="myModalLabel">Служба підтримки</h4>
     <script src="bower_components/chosen/chosen.jquery.js"></script>
     <script src="bower_components/angular-ui-tree/dist/angular-ui-tree.js"></script>
     <script src="bower_components/angular-ui-select/dist/select.js"></script>
+    <script src="../bower_components/angular-xeditable/dist/js/xeditable.min.js"></script>
```

Comment:
Incorrect link to  bower_components directory. Should be 'bower_components/' instead of '../bower_components/'